specification/automation/resource-manager/Microsoft.Automation/2017-05-15-preview/sourceControl.json

```diff
@@ -0,0 +1,317 @@
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../2015-10-31/definitions.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "name": "automationAccountName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The automation account name."
+          },
+          {
+            "name": "sourceControlName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of source control."
+          },
+          {
+            "$ref": "../2015-10-31/definitions.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../2015-10-31/definitions.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "./definitions.json#/definitions/SourceControl"
+            }
+          },
+          "default": {
+            "description": "Automation error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../2015-10-31/definitions.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Automation/automationAccounts/{automationAccountName}/sourceControls": {
+      "get": {
+        "tags": [
+          "SourceControl"
+        ],
+        "operationId": "SourceControl_ListByAutomationAccount",
+        "description": "Retrieve a list of source controls.",
+        "externalDocs": {
+          "url": "http://aka.ms/azureautomationsdk/sourcecontroloperations"
+        },
+        "x-ms-examples": {
+          "List sourceControls": {
+            "$ref": "./examples/sourceControl/getAllSourceControls.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../2015-10-31/definitions.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "name": "automationAccountName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The automation account name."
+          },
+          {
+            "name": "$filter",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "The filter to apply on the operation."
+          },
+          {
+            "$ref": "../2015-10-31/definitions.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../2015-10-31/definitions.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "./definitions.json#/definitions/SourceControlListResult"
```

Comment:
SourceControlListResult appears nowhere in this PR.

Reply:
@fearthecowboy : Hello, I have a new PR #2158 with the fixes, and I am canceling this one. While I am on vacation, @vrdmr and @mariaromero will follow-up on this. Thanks!